lexibank_northeuralex.py

```diff
@@ -36,43 +35,31 @@ def split_forms(self, row, value):
         """
         return BaseDataset.split_forms(self, row, value)[:1]
 
-    def cmd_install(self, **kw):
+    def cmd_makecldf(self, args):
         def concept_id(s):
-            return slug(s.replace(":", "_").replace("[", "-").replace("]", "-"), lowercase=False)
+            return slug(
```

Comment:
I'd propose to drop that line. For the concept ID, use the concepticon concept list. 

```python
concept_lookup = add_concepts(id_factory=lambda x: x.id.split('-')[-1]+'_'+slug(x.english),
    lookup_factory= 'Name')
```

will need to see what you want as lookup.

Reply:
Yes, better than calling `concept_id` for each row.

Reply:
But how would like to perform the lookup when adding forms? With this factory we map the English name to an ID, but the English name is not found in the lexeme data -- we have the NELEX ids of German gloss plus part of speech (like `Auge:N`) which is the concept list but not in the attributes available for the factory (here: https://github.com/lexibank/pylexibank/blob/be583df1f58c19d0f7a45568c7774f5cffd4cac5/src/pylexibank/cldf.py#L304)

Reply:
So, we had a long thread on this in pylexibank, see [here](https://github.com/lexibank/pylexibank/issues/136). 

The answer is: if you cannot do it, you need to manually go for the lookup. The workaround is defining your own `CustomConcept`, if the concept list on concepticon has the `NelexId`. Then, you add all concepts as once, and instead of `Name`, you pass `NelexID` or how you call it in your `CustomConcept` class. 

If you see any of the datasets I crossed off, like `allenbai`, you can see how I addressed this there: I do it with `writer.add_concept`, as I can't handle the namespace otherwise.